x-pack/plugins/ml/common/constants/data_frame_analytics.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const ANALYSIS_CONFIG_TYPE = {
+  OUTLIER_DETECTION: 'outlier_detection',
+  REGRESSION: 'regression',
+  CLASSIFICATION: 'classification',
+} as const;
```

Comment:
We have the same as an `enum` already in `plugins/ml/public/application/data_frame_analytics/common/analytics.ts`. This one using the `const` approach as well as the location here LGTM so suggest to try to get rid of the on mentioned in this comment.

Reply:
Went through and updated the usage of enum ANALYSIS_CONFIG_TYPE to const ANALYSIS_CONFIG_TYPE here https://github.com/elastic/kibana/pull/76735/commits/dd3517b44446099977761548fc7adf3f0fd4ab6c